keyboards/sevencrumbs/matrix/info.json

```diff
@@ -0,0 +1,47 @@
+{
+	"keyboard_name": "Matrix",
+	"manufacturer": "Seven Crumbs",
+	"url": "https://zoid.com.au/matrix/",
+	"maintainer": "ZoidTechnology",
+	"bootloader_instructions": "Press and hold all four buttons while connecting the USB cable.",
+	"processor": "RP2040",
+	"board": "GENERIC_RP_RP2040",
+	"bootloader": "rp2040",
+	"features": {
+		"bootmagic": true,
+		"extrakey": true,
+		"mousekey": true,
+		"rgb_matrix": true
+	},
+	"usb": {
+		"vid": "0x2E8A",
```

Comment:
Theis appears to be the Raspberry Pi foundation Vendor ID.  Do you have authorization/release to use it.  If so, could you verify that?

Reply:
I've converted indentation to spaces and used `qmk format-json` on `info.json`. The `-i` flag was giving me the error: `qmk: error: unrecognized arguments: -i`. Perhaps something to be looked into? Although there's a good chance I've botched something with my install.

I haven't used `qmk format-c` yet, as it seemed to be doing more harm than good. I can give it another go after spraying a few `// clang-format off` comments if you'd like? It seems to have an insatiable desire to pile as much into a single line as possible.

I was allocated the PID `0x1069` under Raspberry Pi's VID. I don't believe the table has been updated, but I have had email confirmation of the allocation from Raspberry Pi.

Reply:
Your repo is not up to date.

Reply:
Sorry! It should be up to date now.